File: src/main/java/org/demoth/aworlds/server2/model/Cell.java
package org.demoth.aworlds.server2.model;

import java.util.ArrayList;
import java.util.Arrays;
import java.util.Collection;

public class Cell {
    Collection<Actor> actors = new ArrayList<>();

    public Cell(Actor... actors) {
        this.actors.addAll(Arrays.asList(actors));
    }
}

File: src/main/kotlin/org/demoth/aworlds/server2/model/Cell.kt
package org.demoth.aworlds.server2.model

import java.util.ArrayList
import java.util.Arrays

class Cell(vararg actors: Actor) {
    internal var actors: MutableCollection<Actor> = ArrayList()

    init {
        this.actors.addAll(Arrays.asList(*actors))
    }
}
